src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java

```diff
@@ -547,6 +548,7 @@ private void setStringValue(@Nonnull String value)
         private final String name, description;
         private final int type;
         private final boolean required;
+        private final Set<ChannelType> channelTypes;
```

Comment:
EnumSet?

Reply:
can't be an `EnumSet` because of what `unmodifiableSet` returns

Reply:
Not sure if it matters too much as JDA doesn't make EnumSets unmodifiable anywhere that it returns them. Many places _should_ be returning unmodifiable EnumSets but aren't.

I think we probably should prefer EnumSet here and perhaps consider introducing our own implementation of an unmodifiable EnumSet.